.circleci/config.yml

```diff
@@ -1979,3 +2003,16 @@ workflows:
           context:
             - oplabs-gcr
             - slack
+
+  scheduled-bindgen:
+    when:
+      equal: [ build_daily, <<pipeline.schedule.name>> ]
+    jobs:
+      - bindgen-remote:
+          context:
+            - slack
+            - oplabs-etherscan
```

Comment:
Still learning circleci, so a few questions here for my own understanding:
1. What are both of these contexts for
2. What channel are failures posted to?
3. Which part of this config specifies to only notify slack on failures?

Reply:
1. I'm assuming `slack` is what gives CircleCi an API key to post to Slack. `oplabs-etherscan` is giving the Go tests access to the Etherscan and RPC API keys
2. I'm guessing `notify-ci-failures`
3. [This](https://github.com/ethereum-optimism/optimism/blob/b5421fe308bd1c3aef48a587b1f7470d1f348ac4/.circleci/config.yml#L69-L80) is the config for notifying slack

Reply:
2. Confirmed, visiting `https://app.slack.com/client/TRBABKTE0/C03N11M0BBN` takes you to that channel